### DISCUSSION TITLE
Disabling skipped notifications

### DISCUSSION THREAD

**Discussion by danquack** [fluxcd/flux2#3831] [Q&A] [answered]
In this [commit](https://github.com/fluxcd/pkg/commit/51ed871538578d606d1ab299ab5277a45b6ca88a), a new skipped status was added. As a result of updating, and having some manifest disabled from reconciled, our notification channels are filled with <x> skipped. Is there a way to not expose those? I've tried to set "*skipped" as an exclusionList item within the Alert CRD, and it doesn't seem to work.

**Comment by kingdonb** [comment, marked as answer]
Are these resources annotated with the `reconcile: disabled` flag that prevents Kustomization Controller from updating them?

I haven't seen this yet, but I can try to reproduce it. I have a `Receiver` reconciled normally by the flux-system kustomization, and adding the annotation to disable reconciling, I do indeed get this annoying thing in my notifications:

<img width="506" alt="image" src="https://user-images.githubusercontent.com/3286998/234942809-56b86e4b-b5ad-41ba-a570-830a6d278607.png">

We think this log should probably be a `trace` event, and it's probably misclassified. I tried to add my own exclusion so it would not echo in the notifications, here's what I did (and it worked)

```diff
--- a/clusters/moo-cluster/flux-system-extras/on-call-webapp-alert.yaml
+++ b/clusters/moo-cluster/flux-system-extras/on-call-webapp-alert.yaml
@@ -10,6 +10,7 @@ spec:
   summary: Internal (staging) cluster ROOT events
   exclusionList:
     - '^Dependencies do not meet ready condition, retrying in \d+s$'
+    - '.*skipped$'
   eventSources:
     - kind: GitRepository
       name: '*'
```

Please note the regex: `.*skipped$` begins with `.` since it is a regex, the `*` in isolation doesn't mean anything. `*` is regex for zero or more, but it needs to refer to something. The `.` is the wildcard. Hope this helps!
